src/containers/Error/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
```

Comment:
This component is awesome! What I'd recommend, because I assume we'd want to reuse it for other errors, is having `:errorTitle` be a key to return text given THAT error. For example, `/errors/out-of-range` would then return `'s an invalid Surah/Ayah. Please go to the <a href="/">home page </a>and select a surah'` and say `/errors/some-other-error` would return that text.